x-pack/legacy/plugins/lens/public/metric_visualization_plugin/metric_expression.tsx

```diff
@@ -106,18 +106,7 @@ export function MetricChart({
   }
 
   return (
-    <VisualizationContainer
-      reportTitle={title}
-      style={{
-        display: 'flex',
-        flexDirection: 'column',
-        justifyContent: 'center',
-        alignItems: 'center',
-        maxWidth: '100%',
-        maxHeight: '100%',
-        textAlign: 'center',
-      }}
-    >
+    <VisualizationContainer reportTitle={title} className="lnsMetricContainer">
```

Comment:
Just to nit pick, the appropriate BEM naming should be 

```suggestion
    <VisualizationContainer reportTitle={title} className="lnsMetricExpression__container">
```